quic/s2n-quic-core/src/recovery/bbr/snapshots/quic__s2n-quic-core__src__recovery__bbr__pacing__events__one_rtt.snap

```diff
@@ -2,4 +2,4 @@
 source: quic/s2n-quic-core/src/recovery/bbr/pacing.rs
 expression: ""
 ---
-PacingRateUpdated { path_id: 0, bytes_per_second: 2747252, burst_size: 12000, pacing_gain: 2.77 }
+PacingRateUpdated { path_id: 0, bytes_per_second: 2742301, burst_size: 12000, pacing_gain: 2.77 }
```

Comment:
I see that all of these estimates are rounding down. This one is the most extreme with ~5000 bytes. I'm not sure it's a huge cause for concern but it _could_ mean that we're not sending as fast since we're underestimating the bandwidth, right?

Reply:
Yeah we were overestimating before. The math previously would be
```
(1000/ 2.77)/.99 = 364.657 nanos/byte, which truncates to 364 nanos/byte = 2747252 bytes/sec
```

Now the math is:
```
1000 * 1024 / 2.77 / .99 = 373409.182 nanos/kibibyte, which truncates to 373409 nanos/kibibyte = 2742301 bytes/sec
```